Makefile

```diff
@@ -127,6 +127,8 @@ godoc: install
 # --- frontend -----------------------------------------------------------------
 NODEPREFIX = .hermit/node
 NODELIB = $(NODEPREFIX)/lib
+# TODO: BASEURL should be calculated dynamically for CI/PR.
+export BASEURL ?= https://evy.dev
```

Comment:
I would add a comment as to why this is exported:

    # BASEURL needs to be in the environment so that `e2e/playwright.config.js` can see it
    # when the `e2e` target is called.

or something like that.

Reply:
ok, done. thanks.